lenses/connect-customlens-globalResiliency.json

```diff
@@ -796,23 +737,35 @@
 				},
 				{
 					"id": "GlobalResiliency_12",
-					"title": "How are you ensuring configuration parity between your two instances?",
-					"description": "To accommodate your chosen multi-region strategy, a deployment methodology needs to be defined for your in-scope services.",
+					"title": "How are you ensuring configuration parity between your two Amazon Connect instances?",
+					"description": "To accommodate your chosen Global Resiliency strategy, a deployment methodology needs to be defined for your in-scope services.",
 					"choices": [
 						{
 							"id": "choice1",
+							"title": "ACGR Global Configuration Management.",
+							"helpfulResource": {
+								"displayText": "ACGR Global Configuration Management mirrors resources across AWS Regions, including Flows, Flow modules, Users, Routing Profiles, for a complete list access the the More Information link below.",
+								"url": "https://docs.aws.amazon.com/connect/latest/adminguide/create-replica-connect-instance.html#mirrored-resources"
+							},
+							"improvementPlan": {
+								"displayText": "ACGR instances onboarded from Nov-2023 have Global Configufation Management enabled by default, previous customers will need to follow a procedure to enable it.",
+								"url": "https://docs.aws.amazon.com/connect/latest/adminguide/get-started-connect-global-resiliency.html"
+							}
+						},						
+						{
+							"id": "choice2",
 							"title": "Manual Backup and Restore.",
 							"helpfulResource": {
-								"displayText": "Amazon Connect and other AWS services provide manual back and restore functionality for migrating configurations across environments.",
+								"displayText": "If ACGR Global Configuration Management is not enabled (pre Nov-2023), Amazon Connect and other AWS services provide manual back and restore functionality for migrating configurations across environments.",

```

Comment:
Where is this pre-Nov 2023 date communicated publicly? If this isn't documented, we shouldn't include it as these lenses are not designed to be the source of truth

Reply:
Yes, it does, it is in our Release Notes